vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:error="error"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
+				<p class="wikit-DateInput__IntroText">
+					{{ resultsIntroText }}
+				</p>
+				<p class="wikit-DateInput__ParsedValue" v-if="parsedValue">
+					<b>{{ parsedValue }}</b>
+				</p>
+				<slot v-if="!value" name="prompt" />
```

Comment:
Not blocking this PR: But since we now show the intro text always, do we want the prompt to actually still be a slot? That is where the application can inject whatever markup it wants. Or would we not prefer for this now to be a text-only prop? @SaiSan-WMDE 

Though, this is not blocking the merge and can still be adjusted later on.

Reply:
@Ladsgroup and I just discussed this and originally agreed to keep the slot. But you're right, this is some static text that will always be displayed, so using a text-only prop like in the other cases makes more sense and is more consistent?

Reply:
You tell me 🙃
This really depends on what you envision might be inserted there by the application.
If it is only ever going to be plain text that is always supposed to be styled the same way, then a prop makes more sense.
On the other hand, if you want to keep the possibility open that the application might want to provide some copy that includes a "learn more" link, then we should keep the slot.

If you two discussed this and decided that to keep the slot, then I'm confident that this is the right decision and am very happy to approve this as soon as we have the note in the CHANGELOG.md :)

Reply:
I believe this should be a text slot for now, since I cannot foresee any use cases that justify the use of a slot at the moment. A slot could be introduced later if that changed.

Reply:
Implemented as requested.